### PR TITLE
Follow

### PR DESCRIPTION
**Pull request by Kluckmuck**

### CODE REVIEW

File: backend/trips/models.py [hunk 1]
```diff
@@ -21,8 +21,13 @@ class Comment(models.Model):
     content = models.CharField(max_length=120)
     date = models.DateField(auto_now_add=True)
 
+class Follow(models.Model):
+    target_id = models.ForeignKey(User, related_name='follow_target', on_delete=models.CASCADE)
```
Comment (Kanabenki) [inline backend/trips/models.py]: Isn't it supposed to be target and

Comment (Kluckmuck) [reply backend/trips/models.py]: I wanted to be consistent with the rest of our fields. But you are right, we get the object and not the ID. This was the way the TA setup their models in HWs before, which is why it stayed like this. To get the actual ID you would have to do target_id.id, that I agree is somewhat confusing.

Comment (Kluckmuck) [reply backend/trips/models.py]: So either we change all models, or we keep it like this imo.

Comment (Kanabenki) [reply backend/trips/models.py]: Ok then let's merge it as is, and fix everything once while writing the angular services, it is just a matter of renaming as we write the corresponding api requests in frotend

**Pull request merged**